src/types.ts

```diff
@@ -9,17 +9,19 @@ import {
 } from './constants'
 import { DEFAULT_THEMES } from './defaultThemes'
 
-export type DataTheme = typeof DEFAULT_THEMES[number] | string
+export type DataTheme = (typeof DEFAULT_THEMES)[number] | string
 
 export interface IComponentBaseProps {
   dataTheme?: DataTheme
 }
 
-export type ComponentColor = typeof componentColors[number]
+export type ComponentColor = (typeof componentColors)[number]
```

Comment:
Are these extra () needed?

Reply:
4f01cbe1bc2a1b9371cf627bd3688683bdf214bd Fixed in this commit.